includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -527,4 +527,25 @@ public function payment_fields() {
 			<?php
 		}
 	}
+
+
+	/**
+	 * Returns the list of available payment method types for UPE.
+	 * See https://stripe.com/docs/stripe-js/payment-element#web-create-payment-intent for a complete list.
+	 *
+	 * @return string[]
+	 */
+	public function get_upe_available_payment_methods() {
+		$methods = parent::get_upe_available_payment_methods();
+
+		$methods[] = 'giropay';
```

Comment:
Now: enabling `giropay` and `sofort` in UPE (other payment methods can be added by the dev tools)